### PR TITLE
Fix field names in Advanced logging examples

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
- 0bf2517ea480f5467fcb856e0c9c402fb482ae52: Fix a couple of wrong field names like using `MaxSizeMB`, while `max_size` is correct.
- cbf44bd3d4544d57289e1d23e87146d5058b2dd1: Documented `enable_caller` field
- 03b30f478768c7ebe976b6d1ede54998af662315: The `tag` filed belongs to the Syslog config, not TCP config.
- 5a270b8d34a72c6f850d8ac82a3c3b30e47e69c4: Format tables

### CODE REVIEW

File: source/manage/logging.rst [hunk 1]
```diff
@@ -293,83 +295,87 @@ Plain log format configuration options
...
+| **ID** | **Name**              | **Description**                                                        |
++--------+-----------------------+------------------------------------------------------------------------+
+| 100    | ``audit-api``         | API events                                                             |
++--------+-----------------------+------------------------------------------------------------------------+
+| 101    | ``audit-content``     | Content changes. This log level can generate considerably more records |
+|        |                       | than the other audit log levels.                                       |
++--------+-----------------------+------------------------------------------------------------------------+
+| 102    | ``audit-permissions`` | Permission changes                                                     |
++--------+-----------------------+------------------------------------------------------------------------+
+| 103    | ``audit-cli``         | CLI operations                                                         |
```
[the remark this answers is not in the document]
Comment (wiggin77) [reply source/manage/logging.rst:366]: Can we add a column or something to help make it more clear?

File: source/manage/logging.rst [hunk 2]
```diff
@@ -411,20 +417,22 @@ Syslog target configuration options
...
+| tls      | bool     | Create a TLS connection to the server receiving the log records. Default is ``false``.                                   |
++----------+----------+--------------------------------------------------------------------------------------------------------------------------+
+| cert     | string   | Path to a cert file (.pem) to be used when establishing a TLS connection to the server.                                  |
++----------+----------+--------------------------------------------------------------------------------------------------------------------------+
+| insecure | bool     | Mattermost accepts any certificate presented by the server, and any host name in that certificate. Default is ``false``. |
+|          |          | **Note**: Should only be used in testing environments, and shouldn’t be used in production environments.                 |
++----------+----------+--------------------------------------------------------------------------------------------------------------------------+
+| tag      | string   | Syslog tag field.                                                                                                        |
```
Comment (wiggin77) [inline source/manage/logging.rst:434]: Nice catch!

**Review by wiggin77 — APPROVED** [review]
Nice improvements!  :+1:

File: source/manage/logging.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (hanzei) [reply source/manage/logging.rst:366]: We could also split the table in two: one for audit logs and one for server (?) logs.

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**